### DISCUSSION TITLE
Awkward Scrolling

### DISCUSSION THREAD

**Discussion by NoahSimonCDI** [ITPNYU/booking-app#245] [General]
When a space and day is selected, there is something confusing going on with scrolling. There ends up being two scroll bars, one for the embedded google calendar, the other for the page itself. If scrolled all the way to the end of the day on the calendar section, the main portion of the page seems to not be able to scroll. So you have to scroll the calendar back up towards 1am, and then you can scroll the page which allows you to continue the process. This is more confusing in Safari, where (may be my settings) the scroll bars only appear when scrolling. In Chrome, this is easier to see and hence navigate. But definitely look at this in different browsers, as it may look and behave differently in each.

**Comment by ecb344** [comment]
I agree with Noah here, it is not easy to navigate the booking calendar. In addition to the scrolling difficulty, it is also hard to determine availability because the embedded section is quite small compared to the calendar its self (vertically). Is it possible to make the embedded section larger (vertically) so that one can scroll the entire page down (seeing less of the banner) to see more of the calendar?

![image](https://github.com/ITPNYU/mc-reservation-tool/assets/173960321/8ba92bec-520b-4c55-b480-f3f79c411614)

**Comment by nopivnick** [comment]
cc: @ss12430 

thanks for your feedback. these are issues with the Media Commons website design, not the booking tool, and so out of scope for this project.

that said, it should be possible to expand the amount of space dedicated to the embed, yes. furthermore, the UX/UI would be better if it were a full page embed rather than an insert on the page.

**Reply by ecb344** [reply]
Hey Noah - copy that! I didn't read Samantha's email closely enough to see that this github is only for the booking tool.

I will speak with the website team.

Thanks,
Erik
